pkg/synchronization/controller.go

```diff
@@ -534,6 +539,38 @@ func (c *Controller) createOrUpdateSecret(logger *zap.Logger, secretSpec *core_v
 	return true, secret, err
 }
 
+func (c *Controller) ensureCommonSecretExists(logger *zap.Logger, ns *core_v1.Namespace) (bool /* retriable */, error) {
+	_, err := c.MainClient.CoreV1().Secrets(ns.Name).Get(apik8scompute.CommonSecretName, meta_v1.GetOptions{})
```

Comment:
Use package level constant you defined =) Here are in other places.

Reply:
🤦‍♂️ Updated